src/hacbs/components/Commits/tabs/CommitsOverviewTab.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import { Commit } from '../../../types';
+
+type ApplicationOverviewTabProps = {
+  commit: Commit;
+};
+
+const ApplicationOverviewTab: React.FC<ApplicationOverviewTabProps> = ({ commit }) => {
+  return <>Overview of {commit.sha.slice(0, 7)}</>;
```

Comment:
Create a utility function for creating a short display sha and reuse it everywhere.

Reply:
done